### PR TITLE
[HttpClient] Mention `extra.trace_content` option

### LINKED ISSUE #14424: [HttpClient] added `extra.trace_content` option to `TraceableHttpClient…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#38587
| PR author(s) | @nicolas-grekas
| Merged in    | 5.x

### PR DESCRIPTION
**Pull request by MatTheCat**
Resolves https://github.com/symfony/symfony-docs/issues/14424

### CODE REVIEW

**Comment by MatTheCat** [comment]
Uh should I really rebase it on 5.4? The documentation (and PR placeholder) says

> If you are […] documenting a new feature, switch to the first Symfony version that included it.

:thinking:

**Comment by OskarStark** [comment]
> Uh should I really rebase it on 5.4?

Yes please, as 5.2 is not maintained anymore

**Comment by MatTheCat** [comment]
Okay, done :ok_hand:

File: http_client.rst [hunk 1]
```diff
@@ -814,6 +814,21 @@ ensure local networks are made inaccessible to the HTTP client::
     // but all the other requests, including other internal networks, will be allowed
     $client = new NoPrivateNetworkHttpClient(HttpClient::create(), ['104.26.14.0/23']);
 
+Profiling
+~~~~~~~~~
+
+When you are using the :class:`Symfony\\Component\\HttpClient\\TraceableHttpClient`
+responses content will be kept in memory, which can cause different issues.
```
Comment (OskarStark) [inline http_client.rst]:
```suggestion
responses, content will be kept in memory, which can cause different issues.
```

File: http_client.rst [hunk 2]
```diff
@@ -814,6 +814,21 @@ ensure local networks are made inaccessible to the HTTP client::
...
+
+You can disable this behavior by setting the ``extra.trace_content`` to ``false``
```
Comment (OskarStark) [inline http_client.rst]:
```suggestion
You can disable this behavior by setting the ``extra.trace_content`` option to
```

File: http_client.rst [hunk 3]
```diff
@@ -814,6 +814,21 @@ ensure local networks are made inaccessible to the HTTP client::
...
+in your requests’ options::
```
Comment (OskarStark) [inline http_client.rst]:
```suggestion
``false`` in your requests’ options::
```

File: http_client.rst [hunk 4]
```diff
@@ -814,6 +814,21 @@ ensure local networks are made inaccessible to the HTTP client::
...
+
+    $response = $client->request('GET', 'https://...', [
+        'extra' => ['trace_content' => false],
+    ]);
+
+This setting won’t have any effect with other clients.
```
Comment (OskarStark) [inline http_client.rst]:
```suggestion
This setting won’t have any effect with other clients.

.. versionadded:: 5.4

    The ``extra.trace_content`` option was introduced in Symfony 5.2.
```

**Comment by MatTheCat** [comment]
@OskarStark thanks for the review :pray: 

I changed things a little further; I hope it’s okay.

**Comment by OskarStark** [comment]
Thank you @MatTheCat.

**Pull request merged**